lib/uwb/protocols/fira/UwbConfiguration.cxx

```diff
@@ -18,6 +18,116 @@ const std::unordered_set<ResultReportConfiguration> UwbConfiguration::ResultRepo
     ResultReportConfiguration::AoAAzimuthReport
 };
 
+/**
+ * @brief map of uci tag to a function of how to generate that uci tag given UwbConfiguration
+ *
+ */
+const std::unordered_map<uwb::protocol::fira::UwbApplicationConfigurationParameterType, std::function<std::optional<uwb::protocol::fira::UwbApplicationConfigurationParameterValue>(UwbConfiguration*)>> uciGenerators{
```

Comment:
Yea I'm actually considering if it might be more appropriate to move the oob2uci translation to a separate header, one that only UwbSession includes, so that way UwbConfiguration doesn't have to know about the uci type and vice versa

Reply:
I just pushed the change that moves the translation function to UwbSession.hxx instead of being tied to UwbConfiguration

Reply:
It's now in its own header, located in UwbOobConversion.hxx